statsmodels/discrete/tests/test_truncated_model.py

```diff
@@ -45,9 +51,12 @@ def test_bic(self):
 
     def test_fit_regularized(self):
         model = self.res1.model
-
         alpha = np.ones(len(self.res1.params))
-        res_reg = model.fit_regularized(alpha=alpha*0.01, disp=0)
+
+        with warnings.catch_warnings():
```

Comment:
Any reason not to use `with pytest.warns(UserWarning)` to simplify?

Reply:
It shouldn't raise a warning. This is just to reduce a bit of test noise.
fit_regularized failures should be investigated and cleared up, but I never took the time for it for the new count models.